### PR TITLE
Add popup menu that like `City Guides by National Geographic`

### PR DESCRIPTION
**Pull request by troyz**
This is a popup menu that like `City Guides by National Geographic`, the document have updated, please reference the section `Circle popup menu with NAPinAnnotation`.

Thanks!

### CODE REVIEW

File: NAMapKit.podspec [hunk 1]
```diff
@@ -13,4 +13,5 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.dependency 'SDWebImage'
   s.dependency 'ARTiledImageView'
+  s.dependency 'pop'
```
Comment (dblock) [inline NAMapKit.podspec]: This dependency should not be added to NAMapKit, but to the demo project, right?

**Comment by dblock** [comment]
Same comment as before. To get it merged it would need tests, but you can also add this to the demo project instead of extending the library proper. It looks like you attempted the latter, but do make sure that you're not adding anything to the library itself in that case.

**Comment by orta** [comment]
Wow - this is quite a feature - thanks @troyz 

I'm with @dblock - I'd love to see this added to the example application instead of directly into NAMapKit itself, thanks for contributing back 👍

**Comment by dblock** [comment]
Also, another option would be to make another pod that depends on namapkit, like namapkit-popupmenu. Something we could link from our README and that would be a good extension example! Then you can do whatever you want there @troyz :)

**Comment by troyz** [comment]
Ok, i will add this feature to the example application. @dblock @orta :)

**Pull request closed without merging**